src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/LockFileUtils.cs

```diff
@@ -273,10 +281,13 @@ private static void ApplyAliases(LibraryDependency libraryDependency, LockFileIt
             IReadOnlyList<SelectionCriteria> orderedCriteria,
             ContentItemCollection contentItems)
         {
+            // TODO: does this need the maccatalyst check too?
```

Comment:
I think yes. You can pass the maccatalist object from `AddMSBuildAssets()` method parameter. Just an o

It seems `AddMSBuildAssets()` is called in `RestoreCommand.AddAssets()` which has `MaccatalystFallback` parameter.

Reply:
I agree. All assets selection need to account for fallbacks.

Reply:
I'm not approving because TODOs shouldn't be left in the code like this. Either fix it, or if you determine it's not needed, delete the comment, or replace it with a URL to a NuGet/Home issue tracking the additional work.